### PR TITLE
N28: Added Hydrofoil Generation section

### PR DESCRIPTION
**Pull request by kunos**

### CODE REVIEW

File: content/news/028/index.md [hunk 1]
```diff
@@ -218,6 +218,37 @@ November's full weekly devlogs: "This Week In Veloren...":
 [veloren-reading-club-2]: https://www.youtube.com/watch?v=n8XayRvVBEs
 [veloren-rain-storm]: https://www.youtube.com/watch?v=MZwfaohynvc
 
+### [Hydrofoil Generation]
+
+![hgs_screen](hgs5.png)
+
+[Hydrofoil Generation] ([Facebook](https://www.facebook.com/HydrofoilGenerationSailing),
+[Discord](https://discord.gg/DtKgt2duAy)) is a realistic sailing/foiling
+inshore simulator in development for PC/Steam that will put you in the
```
Comment (AngelOnFira) [inline content/news/028/index.md:227]: Please move all links to the bottom :)

File: content/news/028/index.md [hunk 2]
```diff
@@ -218,6 +218,37 @@ November's full weekly devlogs: "This Week In Veloren...":
...
+driving seat of modern competitive sailing.
+
+November was dedicated to the launch of the [Steam Store][hgs_steam] page and associated
+[trailer][hgs_trailer] showcasing Hydrofoil Generation's custom Rust engine in motion
+for the first time. Your chance to Wishlist a Rust made engine on Steam is now
+a reality.
```
Comment (AngelOnFira) [inline content/news/028/index.md:233]: I'm not sure about this line, as it sort of makes it sound like there aren't other Rust games on steam (afaik there are a few)

File: content/news/028/index.md [hunk 3]
```diff
@@ -218,6 +218,37 @@ November's full weekly devlogs: "This Week In Veloren...":
...
+
+Constant tweaks to the boat behavior and addition of gameplay features are
+ongoing as the planned Q2 2022 Steam Early Access release gets closer
+and closer.
+
+December 2022 will see an attempt to port the rendering backend of the game
```
Comment (AngelOnFira) [inline content/news/028/index.md:239]: December 2021?

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
A few changes, and could you compress the image and change the format to jpeg? Thanks!

File: content/news/028/index.md (hunk 1 above)
Comment (kunos) [reply content/news/028/index.md]: Should be done with my last commit.

File: content/news/028/index.md (hunk 2 above)
Comment (kunos) [reply content/news/028/index.md]: You are right, I took it out.

File: content/news/028/index.md (hunk 3 above)
Comment (kunos) [reply content/news/028/index.md]: Fixed in my last commit

**Review by AngelOnFira — APPROVED** [review]

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
It looks like I don't have write permission to your branch from this PR. Can you move this section down to the bottom of the game section? I like to keep it in order of when items are merged :)

**Review by AngelOnFira — APPROVED** [review]

**Comment by ozkriff** [comment]
A 1.5mb png file seems to slip into the PR - but since it's already merged we can't do much now

**Comment by AngelOnFira** [comment]
Oh, I guess it wasn't removed when the edited version was added, my bad. It shouldn't be bad if we just delete it with a commit though, Since netlify only clones with depth 1 I assume. It does make the repo larger, but since we're not using LFS, this isn't really avoidable :P

**Pull request merged**